contribs/gmf/src/directives/displayquerygrid.js

```diff
@@ -274,7 +274,7 @@ gmf.DisplayquerygridController = function($injector, $scope, ngeoQueryResult, ng
 gmf.DisplayquerygridController.prototype.$onInit = function() {
   this.removeEmptyColumns_ = this['removeEmptyColumnsFn'] ? this['removeEmptyColumnsFn']() === true : false;
   this.maxRecenterZoom = this['maxRecenterZoomFn'] ? this['maxRecenterZoomFn']() : undefined;
-  this.mergeTabs_ = this['mergeTabsFn'] ? this['mergeTabsFn']() : {};
+  this.mergeTabs_ = this['mergeTabs'] ? this['mergeTabs'] : {};
```

Comment:
You can remove this line and change the line 217 from:

````
   /**
    * @type {!gmfx.GridMergeTabs}
    * @private
    */
   this.mergeTabs_ = {};
````

to

````
   /**
    * @type {!gmfx.GridMergeTabs}
    */
   this.mergeTabs = {};
````
Then, adapt all `this.mergeTabs_` to `this.mergeTabs`;